### PR TITLE
[Chore] - Added Redirection Notes

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

This PR adds redirection notes to all the current docs we have

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/book/en/src/develop_plugin/rust-sdk.md [hunk 1]
```diff
@@ -1,49 +1,3 @@
...
-pub fn imshow(window_name: &str, mat_key: u32) -> () {
-    unsafe { generated::imshow(window_name.as_ptr(), window_name.len(), mat_key) }
-}
-pub fn waitkey(delay: u32) -> () {
-    unsafe { generated::waitkey(delay) }
-}
-```
-
-[^1]: <https://github.com/second-state/witc>
+> This part has been moved to <https://wasmedge.org/docs/contribute/plugin/develop_plugin_rustsdk#example-wasmedge_opencvmini>. Please use our new docs.
```
Comment (alabulei1) [inline docs/book/en/src/develop_plugin/rust-sdk.md:3]: The URL should be https://wasmedge.org/docs/contribute/plugin/develop_plugin_rustsdk/ ?

File: docs/book/en/src/use_cases.md [hunk 2]
```diff
@@ -1,19 +1,3 @@
...
-
-* WasmEdge could support [high performance DSLs (Domain Specific Languages) or act as a cloud-native JavaScript runtime](use_cases/js_or_dsl_runtime.md) by embedding a JS execution engine or interpreter.
-
-* Developers can leverage container tools such as [Kubernetes](use_cases/kubernetes.md), Docker and CRI-O to deploy, manage, and run lightweight WebAssembly applications.
-
-* WasmEdge applications can be plugged into existing [application frameworks or platforms](use_cases/frameworks.md).
-
-If you have any great ideas on WasmEdge, don't hesitate to open [a GitHub issue](https://github.com/WasmEdge/WasmEdge/issues) to discuss together.
+> This part has been moved to <https://wasmedge.org/docs/category/use-cases>. Please use our new docs.
```
Comment (alabulei1) [inline docs/book/en/src/use_cases.md:3]: This is not the same thing.

File: docs/book/en/src/use_cases/kubernetes/demo/server.md [hunk 3]
```diff
@@ -1,125 +1,3 @@
...
-
-# You may need to use docker login to create the `~/.docker/config.json` for auth.
-#
-# docker login
-
-sudo buildah push --authfile ~/.docker/config.json http_server docker://docker.io/wasmedge/example-wasi-http:latest
-```
-
-That's it! Now you can try to run it in [CRI-O](../cri/crio.md#run-a-http-server-app) or [Kubernetes](../kubernetes/kubernetes-crio.md#a-webassembly-based-http-service)!
+> This part has been moved to <https://wasmedge.org/docs/develop/rust/socket_networking/server#an-http-server-example>. Please use our new docs.
```
Comment (alabulei1) [inline docs/book/en/src/use_cases/kubernetes/demo/server.md:3]: The source of this part is here: https://github.com/second-state/wasmedge-containers-examples/blob/main/http_server_wasi_app.md

File: docs/book/en/src/use_cases/kubernetes/kubernetes/kubernetes-containerd.md [hunk 4]
```diff
@@ -1,119 +1,3 @@
...
-Since we are using `hostNetwork` in the `kubectl run` command, the HTTP server image is running on the local network with IP address `127.0.0.1`.
-Now, you can use the `curl` command to access the HTTP service.
-
-```bash
-$ curl -d "name=WasmEdge" -X POST http://127.0.0.1:1234
-echo: name=WasmEdge
-```
-
-That's it!
+> This part has been moved to <https://wasmedge.org/docs/category/high-level-runtime-support>. Please use or new docs
```
Comment (alabulei1) [inline docs/book/en/src/use_cases/kubernetes/kubernetes/kubernetes-containerd.md:3]: This part is here: https://wasmedge.org/docs/develop/deploy/kubernetes/kubernetes-containerd-crun

File: docs/book/en/src/write_wasm/python.md [hunk 5]
```diff
@@ -1,45 +1,3 @@
...
-Then you could get a Python shell in WebAssembly!
-
-## Grant file system access
-
-You can pre-open directories to let WASI programs have permission to read and write files stored on the real machine. The following command mounted the current working directory to the WASI virtual file system.
-
-```bash
-wasmedge --dir .:. ./target/wasm32-wasi/release/rustpython.wasm
-```
+> This part has been moved to <https://wasmedge.org/docs/develop/python/hello_world#compile-rustpython>. Please use our new docs
```
Comment (alabulei1) [inline docs/book/en/src/write_wasm/python.md:3]: Please use the entire link: https://wasmedge.org/docs/develop/python/hello_world/

File: docs/book/en/src/write_wasm/rust.md [hunk 6]
```diff
@@ -1,121 +1,3 @@
...
-* [Simple networking socket](rust/networking.md) shows how to create simple HTTP client and server applications using the WasmEdge networking socket Rust SDK.
-* [Simple networking socket in https](rust/networking-https.md) shows how to create simple HTTPS client and server applications using the WasmEdge networking socket Rust SDK.
-* [Non-blocking networking socket](rust/networking-nonblocking.md) shows how to create a high-performance non-blocking networking applications with concurrent open connections using the WasmEdge networking socket Rust SDK.
-* [Server-side rendering](rust/ssr.md) shows how to build an interactive web app with Rust, and then render the HTML DOM UI on the server using WasmEdge. The Rust source code is compiled to WebAssembly to render the HTML DOM in the browser or on the server.
-* [Command interface](rust/command.md) shows how to create native command applications for WebAssembly using the Wasmedge command interface Rust SDK.
+> This part has been moved to <https://wasmedge.org/docs/develop/rust/hello_world#a-simple-main-app>. Please use our new docs.
```
Comment (alabulei1) [inline docs/book/en/src/write_wasm/rust.md:3]: I'm curious why you don't use the entire link.

**Review by alabulei1 — APPROVED** [review]

File: docs/book/en/src/use_cases.md (hunk 2 above)
Comment (adithyaakrishna) [reply docs/book/en/src/use_cases.md:3]: Gotcha, so can we remove this page all together?

**Comment by adithyaakrishna** [comment]
@alabulei1 Could you please review this PR?

File: docs/book/en/src/use_cases.md (hunk 2 above)
Comment (alabulei1) [reply docs/book/en/src/use_cases.md:3]: I think we can have a use case page under https://wasmedge.org/docs/start/overview. What do you think?

Comment (adithyaakrishna) [reply docs/book/en/src/use_cases.md:3]: Hey @alabulei1 
Wont that give us too many use case pages?

Comment (alabulei1) [reply docs/book/en/src/use_cases.md:3]: Hi  @adithyaakrishna 

This use case is to tell people where you can use WasmEdge. You can also use usage instead.

The use case under the embed file is to tell people examples of embedding WasmEdge into a host app. 

You can use different words for this one.

Comment (alabulei1) [reply docs/book/en/src/use_cases.md:3]: Hi @adithyaakrishna  do you have any feedback on this question?

Comment (adithyaakrishna) [reply docs/book/en/src/use_cases.md:3]: Hey @alabulei1 
Apologies for the late response, yeah, it makes sense. I guess, I will proceed with using use-cases itself as usage word  IMO wont make much sense here

**Comment by alabulei1** [comment]
LGTM. @hydai I think we can merge this PR.

**Comment by adithyaakrishna** [comment]
LesssGooo! SHIP IT 🚀

**Review by hydai — APPROVED** [review]

**Pull request merged**